features/all-positions/AllPositions.tsx

```diff
@@ -77,37 +75,40 @@ const AllPositions = () => {
                 <TableCell align="right">Tokens Outstanding</TableCell>
                 <TableCell align="right">
                   Collateral Ratio (using price:{" "}
-                  {latestPrice ? utils.formatEther(latestPrice) : "N/A"})
+                  {latestPrice ? latestPrice : "N/A"})
                 </TableCell>
               </TableRow>
             </TableHead>
             <TableBody>
               {activeEmpSponsors &&
-                Object.keys(activeEmpSponsors).map((sponsor: string) => (
-                  <TableRow key={sponsor}>
-                    <TableCell component="th" scope="row">
-                      {sponsor}
-                    </TableCell>
-                    <TableCell align="right">
-                      {prettyBalance(
-                        activeEmpSponsors[sponsor].lockedCollateral
-                      )}
-                    </TableCell>
-                    <TableCell align="right">
-                      {prettyBalance(
-                        activeEmpSponsors[sponsor].tokensOutstanding
-                      )}
-                    </TableCell>
-                    <TableCell align="right">
-                      {prettyBalance(
-                        getCollateralRatio(
-                          activeEmpSponsors[sponsor].lockedCollateral,
-                          activeEmpSponsors[sponsor].tokensOutstanding
-                        )
-                      )}
-                    </TableCell>
-                  </TableRow>
-                ))}
+                Object.keys(activeEmpSponsors).map(
+                  (sponsor: string) =>
+                    activeEmpSponsors[sponsor] &&
+                    activeEmpSponsors[sponsor].collateral &&
+                    activeEmpSponsors[sponsor].tokensOutstanding && (
```

Comment:
nit: I notice a lot of usage of `activeEmpSponsors[sponsor]` here, maybe an intermediate variable could help?

```ts
const activeSponsor = activeEmpSponsors[sponsor]
```

Reply:
Good call